src/Service/userService.js

```diff
@@ -0,0 +1,99 @@
+import { validateUser, hashPassword } from '../Utils/user.js';
+import UserRepository from '../Repository/userRepository.js';
+
+/**
+ *
+ * @param {Object} user
+ * @returns {Object} new user
+ */
+const createUser = async (user) => {
+  try {
+    const { error } = validateUser(user);
+    if (error) {
+      throw new Error(error.details.map((err) => err.message));
+    }
+
+    const hashedPassword = await hashPassword(user.password);
+
+    const newUser = await UserRepository.create({
+      ...user,
+      password: hashedPassword,
+    });
+    return newUser;
+  } catch (err) {
+    console.log('err', err);
+    throw new Error(err.message);
+  }
+};
+
+/**
+ *
+ * @returns all users data
+ */
+const getAllUsers = async () => {
+  try {
+    const users = await UserRepository.getAll();
```

Comment:
another tweaks, you can just simply `return await UserRepository.getAll();`
when the handler on route call `await service.getAllUsers()` the `UserRepository.getAll()` will automatically called as await